src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/MigrateVisitService.kt

```diff
@@ -79,7 +86,7 @@ class MigrateVisitService(
       saveLegacyData(visitEntity, null)
     }
 
-    sendTelemetryData(visitEntity)
+    sendMigratedTrackEvent(visitEntity, TelemetryVisitEvents.VISIT_MIGRATED_EVENT)
```

Comment:
just checking that VISIT_MIGRATED_EVENT is correct (ie you don't differentiate between synchronised and migrated)

Reply:
yes this is correct, no change here see line 100 old code on the left